internal/scheduler/configuration/configuration.go

```diff
@@ -73,6 +73,10 @@ type MetricsConfig struct {
 	TrackedErrorRegexes []string
 	// Metrics are exported for these resources.
 	TrackedResourceNames []v1.ResourceName
+	// Some resource names aren't generic and can be attached to a specific vendor.
+	// e.g. "nvidia.com/gpu" is a vendor specific resource name.
+	// ResourceRenaming maps these vendor specific resource names to a generic name to be used in the exported metrics.
```

Comment:
```suggestion
	// Optionally rename resources in exported metrics.
	// E.g., if ResourceRenaming["nvidia.com/gpu"] = "gpu", then metrics for resource "nvidia.com/gpu" use resource name "gpu" instead.
	// This can be used to avoid illegal Prometheus metric names (e.g., for "nvidia.com/gpu" as "/" is not allowed).
	// It can also be used to track multiple resources within the same metric, e.g., "nvidia.com/gpu" and "amd.com/gpu".
```